versions.json

```diff
@@ -1,4 +1,4 @@
 [
-  "1.2",
-  "1.1"
+  "1.3",
```

Comment:
Is there a way that we could mark this version as **v1.3-Preview**? There will be Beta and RC releases before the official stable release.